### PR TITLE
[executor-benchmark] Simple script loading benchmark

### PR DESCRIPTION
**Pull request by georgemitenkov**
## Description

A simple script benchmark to track loading & caching costs. For now, it allows to
- Execute no-op scripts
- Still spend some time deserializing and verifying code 

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Move Compiler
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by georgemitenkov** [comment]
* **#15098** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-core/15098?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @georgemitenkov and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: testsuite/module-publish/src/packages/simple/scripts/main.move [hunk 1]
```diff
@@ -0,0 +1,17 @@
+script {
+    // Note: this constant can be replaced in compiled script to make it hash to a different value.
+    const SENDER: address = @0x1;
+
+    fun main(sender: &signer) {
+        // The idea is to to ensure that this script takes some time to be deserialized and verified, but the actual
```
Comment (msmouse) [inline testsuite/module-publish/src/packages/simple/scripts/main.move:6]: how about naming the package accordingly

Comment (georgemitenkov) [reply testsuite/module-publish/src/packages/simple/scripts/main.move:6]: Is is part of `simple` package which contains a `simple` module, so either I need to rename that or create a new package just for script, which seems unnecessary?

Comment (msmouse) [reply testsuite/module-publish/src/packages/simple/scripts/main.move:6]: I see. Didn't realize it's an existing module. Is it even allowed to use a different file name or `fun` name then? `simple::main::main()` doesn't describe what you intend to do at all. But I guess it's no big deal.

File: crates/transaction-generator-lib/src/publishing/publish_util.rs [hunk 2]
```diff
@@ -125,6 +129,23 @@ impl Package {
         Self::Simple(modules, metadata)
     }
 
+    pub fn script(publisher: AccountAddress) -> TransactionPayload {
+        let code = &*raw_module_data::SCRIPT_SIMPLE;
+        let mut script = CompiledScript::deserialize(code).expect("Script must deserialize");
+
+        // Change the constant to the sender's address to change script's hash.
```
Comment (igor-aptos) [inline crates/transaction-generator-lib/src/publishing/publish_util.rs:136]: Do you want transactions from the same sender to be the same, or do you want every transaction to be different? If later,  you can use random address here instead?

**Review by igor-aptos — APPROVED** [review]

File: testsuite/module-publish/src/packages/simple/scripts/main.move (hunk 1 above)
Comment (georgemitenkov) [reply testsuite/module-publish/src/packages/simple/scripts/main.move:6]: `main` is not a module here, but a script, can call it `script` ... hm.... `main` describes it as an entry point, but maybe we can rename either Move file, or function, good call

**Pull request merged**